scene/main/node.cpp

```diff
@@ -1998,7 +1998,7 @@ void Node::_clean_up_owner() {
 	data.OW = nullptr;
 }
 
-Node *Node::find_common_parent_with(const Node *p_node) const {
+Node *Node::find_common_owner_with(const Node *p_node) const {
```

Comment:
The original method `find_common_parent_with()` is used as a pre-step to find the common owners.

But looking up the common owner along the ownership chain seems to be more efficient, especially when `editable_children` is enabled.